node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/GroupPreparer.java

```diff
@@ -65,9 +65,10 @@ public GroupPreparer(NodeRepository nodeRepository, Optional<HostProvisioner> ho
         boolean dynamicProvisioningEnabled = hostProvisioner.isPresent() && dynamicProvisioningEnabledFlag
                 .with(FetchVector.Dimension.APPLICATION_ID, application.serializedForm())
                 .value();
+        // Do not in-place resize in dynamically provisioned zones
         boolean inPlaceResizeEnabled = enableInPlaceResize
                 .with(FetchVector.Dimension.APPLICATION_ID, application.serializedForm())
-                .value();
+                .value() && !dynamicProvisioningEnabled;
```

Comment:
Unrelated change?

Reply:
Yes, but I too lazy to make a separate PR for that